tutorials/download_model_weights.md

```diff
@@ -0,0 +1,230 @@
+EleutherAI/pythia-160m
+EleutherAI/pythia-410m
+EleutherAI/pythia-1b
+EleutherAI/pythia-1.4b
+EleutherAI/pythia-2.8b
+EleutherAI/pythia-6.9b
+EleutherAI/pythia-12b
+EleutherAI/pythia-70m-deduped
+EleutherAI/pythia-160m-deduped
+EleutherAI/pythia-410m-deduped
+EleutherAI/pythia-1b-deduped
+EleutherAI/pythia-1.4b-deduped
+EleutherAI/pythia-2.8b-deduped
+EleutherAI/pythia-6.9b-deduped
+EleutherAI/pythia-12b-deduped
+databricks/dolly-v2-3b
+databricks/dolly-v2-7b
+databricks/dolly-v2-12b
+togethercomputer/RedPajama-INCITE-Base-3B-v1
+togethercomputer/RedPajama-INCITE-Chat-3B-v1
+togethercomputer/RedPajama-INCITE-Instruct-3B-v1
+togethercomputer/RedPajama-INCITE-7B-Base
+togethercomputer/RedPajama-INCITE-7B-Chat
+togethercomputer/RedPajama-INCITE-7B-Instruct
+togethercomputer/RedPajama-INCITE-Base-7B-v0.1
+togethercomputer/RedPajama-INCITE-Chat-7B-v0.1
+togethercomputer/RedPajama-INCITE-Instruct-7B-v0.1
+tiiuae/falcon-7b
+tiiuae/falcon-7b-instruct
+tiiuae/falcon-40b
+tiiuae/falcon-40b-instruct
+tiiuae/falcon-180B
+tiiuae/falcon-180B-chat
+openlm-research/open_llama_3b
+openlm-research/open_llama_7b
+openlm-research/open_llama_13b
+lmsys/vicuna-7b-v1.3
+lmsys/vicuna-13b-v1.3
+lmsys/vicuna-33b-v1.3
+lmsys/vicuna-7b-v1.5
+lmsys/vicuna-7b-v1.5-16k
+lmsys/vicuna-13b-v1.5
+lmsys/vicuna-13b-v1.5-16k
+lmsys/longchat-7b-16k
+lmsys/longchat-13b-16k
+NousResearch/Nous-Hermes-llama-2-7b
+NousResearch/Nous-Hermes-13b
+NousResearch/Nous-Hermes-Llama2-13b
+meta-llama/Llama-2-7b-hf
+meta-llama/Llama-2-7b-chat-hf
+meta-llama/Llama-2-13b-hf
+meta-llama/Llama-2-13b-chat-hf
+meta-llama/Llama-2-70b-hf
+meta-llama/Llama-2-70b-chat-hf
+google/gemma-2b
+google/gemma-7b
+google/gemma-2b-it
+google/gemma-7b-it
+stabilityai/FreeWilly2
+codellama/CodeLlama-7b-hf
+codellama/CodeLlama-13b-hf
+codellama/CodeLlama-34b-hf
+codellama/CodeLlama-70b-hf
+codellama/CodeLlama-7b-Python-hf
+codellama/CodeLlama-13b-Python-hf
+codellama/CodeLlama-34b-Python-hf
+codellama/CodeLlama-70b-Python-hf
+codellama/CodeLlama-7b-Instruct-hf
+codellama/CodeLlama-13b-Instruct-hf
+codellama/CodeLlama-34b-Instruct-hf
+codellama/CodeLlama-70b-Instruct-hf
+garage-bAInd/Platypus-30B
+garage-bAInd/Platypus2-7B
+garage-bAInd/Platypus2-13B
+garage-bAInd/Platypus2-70B
+garage-bAInd/Camel-Platypus2-13B
+garage-bAInd/Camel-Platypus2-70B
+garage-bAInd/Stable-Platypus2-13B
+garage-bAInd/Platypus2-70B-instruct
+togethercomputer/LLaMA-2-7B-32K
+microsoft/phi-1_5
+microsoft/phi-2
+mistralai/Mistral-7B-v0.1
+mistralai/Mistral-7B-Instruct-v0.1
+mistralai/Mixtral-8x7B-v0.1
+mistralai/Mixtral-8x7B-Instruct-v0.1
+mistralai/Mistral-7B-Instruct-v0.2
+TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T
+TinyLlama/TinyLlama-1.1B-Chat-v1.0
+Trelis/Llama-2-7b-chat-hf-function-calling-v2
+```
+
+&nbsp;
+### 2. Download Model Weights
+
+To download the weights for a specific model, use the `--repo_id` argument. Replace `<repo_id>` with the model's repository ID. For example:
+
+```bash
+python scripts/download.py --repo_id <repo_id>
```

Comment:
The scripts were moved

```suggestion
python litgpt/scripts/download.py --repo_id <repo_id>
```

You'll need to update the rest too

Reply:
Good call. I updated this. I also found some leftovers in the merge docs.